modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,252 @@
+= Configuring Ingress with HTTP Route-Template Custom Resource in Runtime Fabric
+
+Configure ingress support for Mule apps on the Anypoint Runtime Fabric platform using the HTTP route-template custom resource.
+
+== Benefits of Ingress Configuration with the HTTP Route-Template Custom Resource
+
+Current ingress support for Mule apps on the Runtime Fabric platform is confined to https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes ingress] resources, which restrict you from using cloud vendor-native HTTP routing solutions, such as OpenShift Routes on OpenShift clusters, Istio, or Service Mesh. To address this limitation, Runtime Fabric supports a custom resource `httproutetemplates.rtf.mulesoft.com` template.
```

Comment:
"restrict you" -> "restricts usage to"

Reply:
I would also change "Service Mesh" (which isn't a product like the others in the list) to "Kubernetes Gateway API"